.github/workflows/repo-vulnerability.yml

```diff
@@ -0,0 +1,40 @@
+name: Trivy Scanner for vulnerabilities - CI/CD
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
```

Comment:
why only on master? wouldn't we want to protect master?

Reply:
The action is triggered every time there is a push to the main branch or for every pull request. I adopted this behavior from the [Trivy Action](https://github.com/aquasecurity/trivy-action) project, which, in my opinion, makes sense.

To address your questions, you are right, if we are using, for example, the Gitflow branching model, we should consider including the develop branch, release branches, hotfix branches, or feature branches as well. The inclusion depends on the branching model used by the team. I was thinking of the 'trunk' model based on my previous job.